system-apps/password-reset-app/pluginDefinition.json

```diff
@@ -0,0 +1,24 @@
+{
+  "identifier": "org.zowe.zlux.ng2desktop.password-reset",
+  "apiVersion": "1.0.0",
+  "pluginVersion": "1.0.0",
+  "pluginType": "application",
+  "isSystemPlugin":true,
+  "webContent": {
+    "framework": "angular2",
```

Comment:
angular

Reply:
@1000TurquoisePogs Plenty of apps still have "angular2" btw, not sure if that is a big deal